### DISCUSSION TITLE
Ability to pin articles in admin

### DISCUSSION THREAD

**Discussion by N6REJ** [joomla/joomla-cms#43594] [Q&A]
### Is your feature request related to a problem? Please describe.
When you have an article that needs frequent updating or is easy to get lost.

### Describe the solution you'd like
ability to "pin" an article using the same method as "featured" but would only pin it to the list view in admin.

### Additional context
Would be really nice to be able to 'pin' an article in the list view.  When you have a lot of articles with similar names its really easy to lose them and waste time trying to find them.

**Comment by brianteeman** [comment]
That is a very specific request and probably best served by a custom solution.

**Comment by jeckodevelopment** [comment]
What about a module where you filter by tag and you use a specific tag for this?

**Comment by micker** [comment]
You Can Do this with my custom dashboard for joomla (free module)
You Can display a module of feature article for exemple

**Comment by N6REJ** [comment]
> What about a module where you filter by tag and you use a specific tag for this?

nice idea

**Comment by brianteeman** [comment]
Still not something for core!

**Comment by jeckodevelopment** [comment]
> Still not something for core!

I wasn't proposing it as a core feature. But something doable by themselves with a module. :)

**Comment by brianteeman** [comment]
@jeckodevelopment I was responding to this being a feature request for core functionality

**Comment by N6REJ** [comment]
@jeckodevelopment I'll give that at try.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43474">issues.joomla.org/tracker/joomla-cms/43474</a>.</sub>

**Comment by N6REJ** [comment]
I found a solution via https://www.joomill-extensions.com/extensions/custom-quick-icons-module
Works great and does exactly what I was thinking.
![image](https://github.com/joomla/joomla-cms/assets/1850089/da799dd5-3eb1-4136-afaa-782ffc8be18b)
![image](https://github.com/joomla/joomla-cms/assets/1850089/8e36a312-5077-4b69-afa8-c396395f5420)
